interface-elements-for-web/articles/grid/customization-dialog/filtering-page.md

```diff
@@ -4,6 +4,24 @@ author: Anna Kondratova
 legacyId: 118822
 ---
 # Filtering Page
-The "Filtering" page allows you to create and change the filter conditions. For each column, select the criteria operator and specify the operand value.
+The **Filtering** page allows you to create and change filter conditions. 
+
+For each column, you can select the criteria operator and specify the operand value.
+
+* Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-expandfilter.png) to expand a column filter condition.
+* Use ![eud-grid-customizationdialog-close-button](../../../images/grid-customizationdialog-removefilter.png) or clear an operand value, to remove a column filter condition.
+
+![eud-grid-customizationdialog-filteringpage](../../../images/img128925.png)
+
+Click a column criteria operator to change it. Select the new operator from the displayed menu.
```

Comment:
"from the displayed menu" Which displayed menu?

Reply:
The menu appears when a column's criteria operator is clicked. Perhaps:
"Click a column's criteria operator to display a list of available operators. Click a new operator to change the current one."